exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook_requirements.txt

```diff
@@ -7,5 +7,6 @@ jupysql==0.10.10
 sqlalchemy_exasol==4.6.3
 stopwatch.py==2.0.1
 boto3==1.26.163
-exasol-notebook-connector @ git+https://github.com/exasol/notebook-connector.git@main
-pickleshare==0.7.5 # See https://github.com/exasol/ai-lab/issues/291 for details.
\ No newline at end of file
+exasol-notebook-connector @ git+https://github.com/exasol/notebook-connector.git@feature/19_move_slc_notebook
```

Comment:
Needs to be reverted as soon as https://github.com/exasol/notebook-connector/pull/122 was merged!!!